schema-comparer/src/SchemaCompareReporter.ts

```diff
@@ -98,6 +100,43 @@ export abstract class SchemaCompareReporter implements ISchemaCompareReporter {
     }
   }
 
+  private reportEntityClassChanges(map: Map<string, EntityClassChanges>) {
+    if (map.size === 0)
+      return;
+
+    this.reportHeader(SCHEMA_ITEM_GROUP_DEPTH, "Classes");
+    for (const [entityClass, changes] of map) {
+      this.reportClassChange(entityClass, changes);
+
+      let markAsRemoved: boolean | undefined;
+
+      if (changes.schemaItemMissing) {
+        markAsRemoved = true;
+      }
+
+      this.reportEntityMixinChanges(changes.entityMixinChanges, markAsRemoved);
```

Comment:
I've seen that in a couple of places, but if we just need to check if something was missing, we could simplify the code to:
```ts
const markAsRemoved = changes.schemaItemMissing !== undefined;
```
or, in some cases, we might not even need that markAsRemoved variable

```ts
this.reportEntityMixinChanges(changes.entityMixinChanges, changes.schemaItemMissing !== undefined);
```

Reply:
It's a good suggestion. I have updated it now.